src/classes/BDI_DataImportDeleteBTN_CTRL.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>39.0</apiVersion>
```

Comment:
good catch.  ideally they would be 37 to be consistent with the rest of our classes, and to avoid issues when classes call other classes on different versions.